core/src/main/java/com/cloud/agent/api/SecurityGroupRulesCmd.java

```diff
@@ -30,12 +30,13 @@
 import org.apache.log4j.Logger;
 
 import com.cloud.agent.api.LogLevel.Log4jLevel;
+import com.cloud.agent.api.to.VirtualMachineTO;
 import com.cloud.utils.net.NetUtils;
 
 public class SecurityGroupRulesCmd extends Command {
     private static final String CIDR_LENGTH_SEPARATOR = "/";
-    private static final char RULE_TARGET_SEPARATOR = ',';
-    private static final char RULE_COMMAND_SEPARATOR = ';';
+    public static final char RULE_TARGET_SEPARATOR = ',';
```

Comment:
I see that RULE_TARGET_SEPARATOR is not being used on tests. Is that correct? If it is indeed not necessary on tests, it can be set back to private.